server/lib/notifications/agents/webhook.ts

```diff
@@ -28,11 +28,16 @@ const KeyMap: Record<string, string | KeyMapFunction> = {
   media_tmdbid: 'media.tmdbId',
   media_tvdbid: 'media.tvdbId',
   media_type: 'media.mediaType',
+  media_jellyfinMediaId: (payload) =>
+    payload?.media?.jellyfinMediaId
+      ? payload.media.jellyfinMediaId
+      : payload?.media?.jellyfinMediaId ?? '',
```

Comment:
I think it would be cleaner that way:
```suggestion
  media_jellyfinMediaId: (payload) => payload.media?.jellyfinMediaId ?? '',
```

Reply:
Hi,
Sorry, it was not intended. My intention was to use either jellyfinMediaId or jellyfinMediaId4k as the media_jellyfinMediaId and then fallback to an empty string if none of them are set. I have updated this to use that fallback logic now in the PR.

The reasoning behind it is that in the API:s at jellyfin there is no distinction between them and during a jellyseerr fulfilled request either of them would be set. Do you think this makes sense or would you prefer if it would only be jellyfinMediaId?

Reply:
Ah yes ok, it makes sense.